js/ui/grid_core/ui.grid_core.columns_controller.js

```diff
@@ -2864,6 +2864,20 @@ module.exports = {
                     }
                     return -1;
                 },
+
+                /**
+                 * @name GridBaseMethods.getVisibleColumnIndex
+                 * @publicName getVisibleColumnIndex(id)
+                 * @param1 id:number|string
+                 * @return number
+                 */
+                getVisibleColumnIndex: function(id, rowIndex) {
+                    let option = this.columnOption(id),
+                        index = option && option.index;
```

Comment:
Use the next more simple code:

const index = this.columnOption(id, "index");

Reply:
Improved column index getting